scripts/deploy_apphub.sh

```diff
@@ -8,9 +8,16 @@ npm install --global apphubdeploy
 # Create a .apphub credentials file
 echo "{\"appHubId\":\"Z6IwqK52JBXrKLI4kpvJ\",\"appHubSecret\":\"$APP_HUB_SECRET\"}" > .apphub
 
+# Get the most recent PR commit
+SHA=`git rev-list --min-parents=2 --max-count=1 HEAD`
+# Pull the name of the PR out of the auto-generated commit description
+PR_DESC=`git log --format=%B -n 1 $SHA | tail -1`
+# This'll break when we hit 1k PRs, but that's still a while away and an easy fix
+PR_NUM=`git log --format=%B -n 1 $SHA | grep -o '#[0-9][0-9][0-9]' | tail -n 1`
```

Comment:
Yup, the `-E` is for BSD grep to support extended regexp, I believe that even GNU grep secretly supports the `-E` option to be compatible with BSD grep.